experiments/functional/cvr-scaleup/test.yml

```diff
@@ -15,28 +16,31 @@
         - include_tasks: /utils/fcm/update_litmus_result_resource.yml
           vars:
             status: 'SOT'
+        
+        - name: Identify the data consistency util to be invoked
+          template:
+            src: data_persistence.j2
+            dest: data_persistence.yml
+
+        - include_vars:
+            file: data_persistence.yml
+        
+        - name: Record the data consistency util path
+          set_fact:
+            data_consistency_util_path: "{{ consistencyutil }}"
+          when: data_persistence != '' 
 
         - block:
```

Comment:
@w3aman  is this block is required here.

Reply:
No... not required... i removed it !!!